cw-rpm.mk

```diff
@@ -76,12 +76,8 @@ RPM_NAMES ?= $(PKG_NAMES)
 
 RPM_ARCH := $(shell rpmbuild -E %{_arch} 2>/dev/null)
```

Comment:
It's unpleasant that you end up defining RPM_ARCH and RPM_NAMES twice - can you pull this out to a separate file that can then be used by both the move and build files